Add Windows/WSL instructions (#368)
AlexsLemonade/OpenScPCA-analysis: docs/technical-setup/index.md

- [Cloning your fork](./clone-the-repo.md) to your computer
- Setting up [additional dependencies](environment-setup/index.md) on your computer that you'll need to contribute to OpenScPCA

## Special considerations for Windows users

Due to the OpenScPCA project's technical environment needs, Windows users will [need to install and use Windows Subsystem for Linux 2 (WSL2)](./install-wsl2.md).

> [WSL2](https://learn.microsoft.com/en-us/windows/wsl/about) lets you run a separate Linux (Ubuntu) environment on your Windows computer and provides a separate Linux file system that is separate from your regular Windows file system.

Reviewer: I tried separating the two parts here, and not saying "separate" separately in every clause. We should probably link to something about the two file systems...
```suggestion
[WSL2](https://learn.microsoft.com/en-us/windows/wsl/about) lets you run a separate Linux (Ubuntu) environment on your Windows computer.
This includes a separate Linux file system that is mostly distinct from your regular Windows file system.
```
